ci/Dockerfile.circle_build

```diff
@@ -38,3 +38,4 @@ RUN curl https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 > /tmp/cloud
   && sudo mv /tmp/cloud_sql_proxy /usr/local/bin && sudo chmod +x /usr/local/bin/cloud_sql_proxy
 
 RUN sudo apt-get install gradle
+RUN sudo gem install jira-ruby
```

Comment:
We only need this for staging releases. I guess it probably doesn't take very long to install this for all builds, but if it did, we could put this in our circle config instead?